_data-prepper/getting-started.md

```diff
@@ -10,40 +10,32 @@ redirect_from:
 
 Data Prepper is an independent component, not an OpenSearch plugin, that converts data for use with OpenSearch. It's not bundled with the all-in-one OpenSearch installation packages.
 
-If you are migrating from Open Distro Data Prepper, visit the [Migrating from Open Distro]({{site.url}}{{site.baseurl}}/data-prepper/migrate-open-distro/) page. 
+If you are migrating from Open Distro Data Prepper, see the [Migrating from Open Distro]({{site.url}}{{site.baseurl}}/data-prepper/migrate-open-distro/) page. 
+{: .note}
 
 ## 1. Installing Data Prepper
 
-There are two ways to install Data Prepper:
+There are two ways to install Data Prepper: Run the Docker image and build from source.
 
-1. Run the Docker image.
-2. Build from source.
-
-The easiest way to use Data Prepper is by running the Docker image. We suggest that you use this approach if you have [Docker](https://www.docker.com) available.
-
-You can pull the Docker image:
+The easiest way to use Data Prepper is by running the Docker image. We suggest that you use this approach if you have [Docker](https://www.docker.com) available. Do the following: 
```

Comment:
Can we assume that users will know to enter this command in a terminal?

Reply:
We don't typically point that out in our other docs, but happy to be proven wrong.